framework/core/command_buffer.cpp

```diff
@@ -166,6 +167,12 @@ void CommandBuffer::begin_render_pass(const RenderTarget &render_target, const s
 	begin_info.clearValueCount   = to_u32(clear_values.size());
 	begin_info.pClearValues      = clear_values.data();
 
+	// Test the requested render area to confirm that it is optimal and could not cause a performance reduction
+	if (!is_render_size_optimal(begin_info.renderArea))
+	{
+		LOGD("Render target extent is not an optimal size, this may result in reduced performance.");
```

Comment:
Can we find someway to print this just once rather than flooding the log with thousands of these?
Most set-top-box resolutions aren't nice multiples of tile-size, so all I see now is these messages scrolling by.

Reply:
The message will display once now, and the calculation of the optimal render size is fixed as there was a bug in how we made our comparisons. Feel free to test it again and let me know what you think